### DISCUSSION TITLE
TypeError: Cannot read property '0' of undefined     at Object.Kt [as decompressSync]

### DISCUSSION THREAD

**Discussion by christopher-jj-k** [101arrowz/fflate#38] [Q&A]
I have been using fflate to decompress a gzipped json file and this has been working fine but now I moved the project to a new server where I am getting this issue when trying to decompress with fflate.decompressSync I have tried the file on my old server and it works there. I've also made sure to check on the arraybuffer and it looks the same. Why is decompression failing?

**Comment by 101arrowz** [comment]
I can take a look at this later, but could you verify you're passing in a `Buffer` or a `Uint8Array`, not an `ArrayBuffer`? Do `Buffer.from(otherBuf)` or `new Uint8Array(otherBuf)` if it is actually an `ArrayBuffer`. If not, post/send me the code you're running and I'll take a look.

**Reply by 101arrowz** [reply]
Hi @christopher-jj-k, are you still having issues with the library?
